index.bs

```diff
@@ -127,7 +127,7 @@ or a combination of both.
 This specification relies on several other underlying specifications.
 
 : HTML5
-:: The concept of <dfn>origin</dfn> and the <dfn>Navigator/dfn> interface are defined in [[!HTML5]].
+:: The concept of <dfn for='web'>origin</dfn> and the <dfn>Navigator/dfn> interface are defined in [[!HTML5]].
```

Comment:
this definition is not strictly true. unfortunately whatwg (and thus W3C) have plowed ahead on re-defining origin without coordinating with IETF wrt RFC6464.  E.g., see https://lists.w3.org/Archives/Public/ietf-http-wg/2016JulSep/0225.html